specification/security/resource-manager/Microsoft.Security/stable/2021-11-01/alerts.json

```diff
@@ -0,0 +1,1120 @@
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/locations/{ascLocation}/alerts/{alertName}/activate": {
+      "post": {
+        "x-ms-examples": {
+          "Update security alert state on a resource group from a security data location": {
+            "$ref": "./examples/Alerts/UpdateAlertResourceGroupLocation_example.json"
+          }
+        },
+        "tags": [
+          "Alerts"
+        ],
+        "description": "Update the alert's state",
+        "operationId": "Alerts_UpdateResourceGroupLevelStateToActivate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/AscLocation"
+          },
+          {
+            "$ref": "#/parameters/AlertName"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/locations/{ascLocation}/alerts/default/simulate": {
+      "post": {
+        "x-ms-examples": {
+          "Simulate security alerts on a subscription": {
+            "$ref": "./examples/Alerts/SimulateAlerts_example.json"
+          }
+        },
+        "tags": [
+          "Alerts"
+        ],
+        "description": "Simulate security alerts",
+        "operationId": "Alerts_Simulate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/AscLocation"
+          },
+          {
+            "$ref": "#/parameters/AlertSimulatorRequestBody"
+          }
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "original-uri"
+        },
+        "responses": {
+          "204": {
```

Comment:
>204 [](http://example.com/codeflow?start=11&length=3)

Your previous status codes were correct, the change is incorrect. The RPC doesn't allow 204 from POST, and 202 is required for async response.

Reply:
Thank you for this comment, this is not an aync operation and the result is without content.
We wish to return 204.
@mentat9